docs/how-to-guides/others/an-example-procedure-for-adding-a-new-node.md

```diff
@@ -0,0 +1,89 @@
+# An example procedure for adding a new node - Case of localization
```

Comment:
```suggestion
# An example procedure for adding and evaluating a new node 
```
Since this page contains a lot of generally applicable content and describes how to evaluate nodes, how about changing the title to this?